assets/scss/modules/_forms.scss

```diff
@@ -57,4 +58,33 @@
       transform: rotate(45deg);
     }
   }
+}
+
+.holder {
+  &-group {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+  }
+
+  &-size1of1 {
+    flex: 1;
+
+    &:nth-last-child(-n+3) {
```

Comment:
Is this ok? ` (-n+3)`

Reply:
Yes, it's okay. Can you think of something better?